.github/workflows/validate_pull_request_main.yaml

```diff
@@ -3,6 +3,9 @@ on:
   pull_request:
     branches:
       - main
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Interesting, is the idea that multiple e2e tests in flight are putting too much load on dev TDR?